src/org/purefn/starman/jedis.clj

```diff
@@ -92,7 +144,10 @@
 
   (write [this ns k value]
     (with-open [c (.getResource pool)]
-      (.set c (common/full-key ns k) value)))
+      (let [fk (common/full-key ns k)
+            enc (encoder config ns)]
+        (set* c fk (encode enc value))
+        (get-decoded c fk enc))))
```

Comment:
Is it necessary to do a read and round trip through the encoder? If `set*` returns true (assuming it returns a bool or something sensible), then can you just return `value` here?

Reply:
no probably not, I can revert this back to just return the value (which is "OK")

Reply:
This function was passed a `value` arg - maybe just return that?